consensus/propagation/reactor.go

```diff
@@ -65,22 +68,19 @@ type Reactor struct {
 	cancel context.CancelFunc
 }
 
-func NewReactor(self p2p.ID, tracer trace.Tracer, store *store.BlockStore, mempool Mempool, options ...ReactorOption) *Reactor {
-	if tracer == nil {
-		tracer = trace.NoOpTracer()
-	}
+func NewReactor(self p2p.ID, store *store.BlockStore, mempool Mempool, privval types.PrivValidator, options ...ReactorOption) *Reactor {
```

Comment:
probably going to modify this function so that it accepts the chainID, since its effectively required